src/annotations/parseMetadata.ts

```diff
@@ -0,0 +1,62 @@
+import safeEval from 'safe-eval'
+
+/**
+ * Parse metadata annotations.
+ * 
+ * Examples:
+ *
+ * ```
+ * @metadata
+ * @metadata(create:false, create: "test")
+ * ``
+ *
+ * @param {string} namespace
+ * @param {string?} description
+ * @returns {object}
+ */
+export function parseMetadata(namespace: string, description: string): any {
+  if (description) {
+    const start = `@${namespace}`
+
+    let line = description.split('\n').map(line => line.trim())
+      .find(line => line.startsWith(start))
+    if (!line) {
+      return undefined
+    }
+    line = line.substr(start.length).trim()
+    if (line === '') {
+      return true
+    }
+
+    const openingTag = '('
+    const closingTag = ')'
+
+    if (!line.startsWith(openingTag)) {
+      console.error(`Can't parse "@${namespace} annotation: Expected opening tag "${openingTag}"`)
+    }
+
+    const startPosition = description.indexOf(line)
+    const startContent = description.substring(startPosition + 1, description.length)
+    const endPosition = startContent.indexOf(')')
```

Comment:
`The indexOf() method returns the position of the first occurrence of a specified value in a string.`

We usually want the last `)` not first. Without that things will fail ` "conflictStrategy": "customExecutor(value, server)"` etc.

Reply:
I think we want indexOf, because you want to find the _next_ ), not the last.
Having multiple annotations like below will cause it to fetch the wrong last closing tag when parsing "@db".

```
"""
@db(...)
@model(...)
"""
```

> Without that things will fail if we do "conflictStrategy": "customExecutor(value, server)" etc.

If we agree to do JSON parse, this won't matter.